cost-analyzer/values.yaml

```
@@ -1871,6 +1871,12 @@ clusterController:
     repository: gcr.io/kubecost1/cluster-controller
     tag: v0.16.13
   imagePullPolicy: IfNotPresent
+
+  # extraEnv:
+  # - name: EXTRA_ENV_VAR
+  #   value: "extra_env_var_value"
+
```

When defining new configurations in the default helm `values.yaml` file, I think we should aim to have everything clearly defined. Only comment out descriptions or examples. For example, I think it should be the following:

```suggestion
  extraEnv: []
    # - name: EXTRA_ENV_VAR
    #   value: "extra_env_var_value"
```